DuggaSys/microservices/accessedService/addUser_ms.php

```diff
@@ -154,5 +155,7 @@
         	}
 		} // End of foreach user
 	} // End ADD_USER
+
+	$array = retrieveAccessedService($pdo, $debug, $userid, null, $log_uuid, $opt, null);
```

Comment:
The $entries array that shows users in the output is currently showing as empty, this is because the retrieve-service is being called with $courseid = null. If you instead assign it to $cid, users will show up in the output and can then be filtered. Like this:
```
	$array = retrieveAccessedService($pdo, $debug, $userid, $cid, $log_uuid, $opt, null);
```